docs/guides/qiskit-addons-cutting-wires.ipynb

```diff
@@ -0,0 +1,335 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1ce517c0-5dfd-4458-82f6-1a21f248603b",
+   "metadata": {},
+   "source": [
+    "# Get started with circuit cutting using wire cuts\n",
+    "\n",
+    "This guide demonstrates a working example of using wire cuts to get started with the `qiskit-addon-cutting` package. It will cover reconstructing expectation values of a seven-qubit circuit using wire cutting and reducing circuit depth and width using gate cutting.\n",
+    "\n",
+    "A wire cut is represented in this package as a two-qubit [`Move`](../api/qiskit-addon-cutting/instructions-move) instruction, which is defined as a reset of the second qubit the instruction acts on followed by a swap of both qubits. This operation is equivalent to transferring the state of the first qubit to the second qubit, while simultaneously discarding the state of the second qubit (as in, the first qubit ends up in the state $|0\\rangle$).\n",
+    "\n",
+    "The package is designed this way primarily because it is consistent with the way you must treat wire cuts when acting on physical qubits. For example, a wire cut might take the state of physical qubit $n$ and continue it as a physical qubit $m$ after the cut. This choice also has the benefit of allowing you to think of \"instruction cutting\" as a unified framework for considering both wire and gate cuts within the same formalism (since a wire cut is just a cut [`Move`](../api/qiskit-addon-cutting/instructions-move) instruction).\n",
```

Comment:
```suggestion
    "The package is designed to be consistent with the way you must treat wire cuts when acting on physical qubits. For example, a wire cut might take the state of physical qubit $n$ and continue it as a physical qubit $m$ after the cut. You can think of \"instruction cutting\" as a unified framework for considering both wire and gate cuts within the same formalism (since a wire cut is just a cut [`Move`](/api/qiskit-addon-cutting/instructions-move) instruction).\n",
```